aws/resource_aws_ebs_snapshot_import.go

```diff
@@ -0,0 +1,431 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/keyvaluetags"
+	tfec2 "github.com/terraform-providers/terraform-provider-aws/aws/internal/service/ec2"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/tfresource"
+)
+
+func resourceAwsEbsSnapshotImport() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsEbsSnapshotImportCreate,
+		Read:   resourceAwsEbsSnapshotImportRead,
+		Update: resourceAwsEbsSnapshotImportUpdate,
+		Delete: resourceAwsEbsSnapshotImportDelete,
```

Comment:
i wonder if these are needed and can use existing snapshot function similar to AMI resources. see `resource_aws_ami_copy` for instance

Reply:
I took a look a couple days ago and didn't think that it would be worth it to pursue further. There were enough differences that (IIRC) it wouldn't be drop-in.